src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -909,13 +909,11 @@ export default class FilingHistoryList extends Mixins(
       item.availableOnPaperOnly ||
       item.isTypeStaff ||
       item.isFutureEffective ||
-      this.isStatusCorrected(item) ||
       this.isTypeAlteration(item) ||
       this.isTypeCorrection(item) ||
       this.isTypeTransition(item) ||
-      this.isTypeIncorporationApplication(item) || // *** temporary
       // the following corrections are supported:
-      // (this.isTypeIncorporationApplication(item) && !this.isBComp) || // *** temporary
```

Comment:
I restored this temporary change, since Edit UI should not correctly handle the new IA corrections (even though the BE doesn't). This is a tiny bit risky but in Prod there's only ever been 2 IA corrections so I don't think this will get us in trouble.